crates/testing-framework/src/asserts/mod.rs

```diff
@@ -54,6 +54,9 @@ pub enum ServiceDefinition {
     /// Some known service by name: "echo", "unit" (more to follow).
     #[strum_discriminants(strum(serialize = "behaviour"))]
     Behaviour(Behavior),
+    /// Same services as defined by the enum element above with dbg! applied to the arguments.
+    #[strum_discriminants(strum(serialize = "dbg_behaviour"))]
```

Comment:
I think calling it just `dbg` is more ergonomic.

Reply:
But `dbg` tells nothing regarding the actual effect that happens.

Reply:
I also vote for `dbg_behaviour`, it makes more sense for people who not familiar with Rust-specific `dbg` function.